book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals # this is supposed to be array that houses the rentals made by persons
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    # rentals = []
+  end
+
+  def add_rentals=(rental)
+    rental.book = self
+    rentals.push(rental)
+  end
```

Comment:
- [ ] In the add_rentals method, the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Check out the example below:
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```
In Rental class you'll be able to push the created rental itself to the `book.rentals`: 
```rb
class Rental
  attr_accessor :date
  attr_reader :book, :person

  def initialize(date, person, book)
    @date = date
    @person = person
    @book = book
    book.rentals << self
    person.rentals << self
  end
end
```